lnwallet/commitment.go

```diff
@@ -654,25 +654,29 @@ type AuxLeafStore interface {
 	// FetchLeavesFromView attempts to fetch the auxiliary leaves that
 	// correspond to the passed aux blob, and pending fully evaluated HTLC
 	// view.
-	FetchLeavesFromView(prevBlob tlv.Blob, view *HtlcView,
+	FetchLeavesFromView(chanState *channeldb.OpenChannel, prevBlob tlv.Blob,
```

Comment:
Alternatively could just be that special funding blob (the one in the open channel struct), we could also use a type alias here. Unsure though if we _do_ need all that other information or not.

Reply:
We need the local/remote channel config, custom blob, thaw height, initiator and channel type. So it's easiest to just pass in the whole channel state. Not sure what a type alias would give us? Other than perhaps being a bit shorter.